local-config.yml

```diff
@@ -51,6 +51,6 @@ utils:
     mean: 0
   services-state:
     ping: passive
-    flood-ping: passive
+    flood-ping: active
```

Comment:
please make this passive

Reply:
Does this happen automatically, without manual override? Coz, I dont remember changing this.

Reply:
It happens automatically. Still, please make that to `passive` and stage it.